numpy/core/shape_base.py

```diff
@@ -177,6 +177,71 @@ def atleast_3d(*arys):
         return res
 
 
+def atleast_nd(n, *arys):
+    """
+    View inputs as arrays with at least `n` dimensions.
+
+    Parameters
+    ----------
+    n : scalar
```

Comment:
I think the large number of changes in `np.ma` suggest this order is not the right one. I would suggest to instead make `n` a keyword-only argument, and flip the order.